webpack.common.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
                 ]
             },
             {
-                test: /\.(gltf|glb|obj|mtl|fbx|dae)$/,
+                test: /\.(gltf|glb|obj|mtl|fbx|dae|bin)$/,
```

Comment:
I'm not sure if this is the best place to put this, but I needed support for bin files so that I could require('...bin') so the gltf binary is added to the /assets/models/ dir

Reply:
looks like the right place to me!:) thanks, added.